src/store.ts

```diff
@@ -17,6 +18,19 @@ export const isShowingPoints = writable(defaults.showPoints);
 export const initialViewport = writable(defaults.viewport);
 export const navMode = writable(NavMode.autofit);
 
+export const storeApiKeys = writable(getStoreApiKeys());
+storeApiKeys.subscribe((val) => {
+  if (!val) {
+    // reset local storage if user decides not to store API keys
+    localStorage.removeItem('api');
+  }
+  localStorage.setItem('store-api', val.toString());
+});
+export const apiKey = writable(getApiKey());
+apiKey.subscribe((val) => {
+  localStorage.setItem('api', val.toString());
```

Comment:
shouldnt this only set the item if `storeApiKeys` is false or unset?

Reply:
Also fixed, and I added some logic to make sure the API keys are properly set/unset if the user clicks on the checkbox a bunch of times.